netutils/ip.py

```diff
@@ -167,22 +171,43 @@ def cidr_to_netmask(cidr):
     raise ValueError("Parameter must be an integer between 0 and 32.")
 
 
+def cidr_to_netmask6(cidr):
+    """Creates a decimal format of a CIDR value.
+
+    Args:
+        cidr (int): A CIDR value.
+
+    Returns:
+      netmask (str): Decimal format (IPv6) representation of CIDR value.
+
+    Example:
+        >>> from netutils.ip import cidr_to_netmask6
+        >>> cidr_to_netmask6(24)
+        'ffff:ff00::'
+        >>> cidr_to_netmask6(17)
+        'ffff:8000::'
+    """
+    if isinstance(cidr, int) and 0 <= cidr <= 128:
+        return str(ipaddress.IPv6Address(((1 << cidr) - 1) << (128 - cidr)))
+    raise ValueError("Parameter must be an integer between 0 and 128.")
+
+
 def get_all_host(ip_network):
     """Given a network, return the list of usable IP addresses.
 
     Args:
         ip_network (str): An IP network in string format that is able to be converted by `ipaddress` library.
 
     Returns:
-        list: List of usable IP Addresses within network.
+        generator: Generator of usable IP Addresses within network.
```

Comment:
I can go either way on this, performance is obviously better as a generator, but just want to make sure it makes sense for the masses.

Reply:
I can yank this out if you like.